sandbox/handlers/messages.js

```diff
@@ -23,6 +24,16 @@ function getOriginatorOdsCode(req) {
   return odsCode;
 }
 
+function getEmailOverride(req) {
```

Comment:
This should avoid any null reference error
```suggestion
function getEmailOverride(req) {
 return req?.body?.data?.attributes?.recipient?.contactDetails?.email
}
```

Reply:
I copied an existing function which was probably meant for an older version of node. I'll make the change, thank you.

Reply:
(This got marked as resolved automatically)